tests/platform/cloud/DisruptionSequenceGradeConfig.js

```diff
@@ -0,0 +1,28 @@
+/*!
+Copyright 2019 OCAD university
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+
+*/
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii");
+
+fluid.require("%gpii-universal");
+fluid.registerNamespace("gpii.tests.cloud.oauth2.settings");
+
+gpii.tests.cloud.oauth2.settings.config = {
```

Comment:
It seems this file (tests/platform/cloud/DisruptionSequenceGradeConfig.js) can be removed once this config is moved into settingsGetTests and settingsPutTests.

Reply:
The definition of the `gpii.test.disruption.sequenceGrade` grade is different depending on whether the tests are run in a development or a production configuration.

For development, the definition is in terms of the `gpii.test.disruption` marker grade and the `gpii.test.couchSequenceGrade` grade.  For production, it's in terms of the marker grade and `gpii.test.standardServerSequenceGrade`.  The two definitions are provided separately by `tests/platform/cloud/DisruptionSequenceGradeConfig.js` and `tests/production/Common.js` respectively.  There should be no definition of the grade in `gpii/node_modules/testing/src/CloudBasedOAuth2TestsUtils.js` since those utilities are shared between the two configurations.

There may be a better way of doing this though.  I am worried that it's not obvious that the definition of the grade is dependent on the configuration, and the fact that you missed it confirms that.   Please advise if there is a better way.

Also, the configuration itself, `gpii.tests.cloud.oauth2.settings.config`, is identical for both SettingsGetTests.js and SettingsPutTests.js.  My thinking is that it's another common data structure that need not be repeated in multiple files.  As such, I put it in a common place.

Reply:
Thanks for the clarification. Let's keep as is.